proc/proc_test.go

```diff
@@ -500,7 +500,7 @@ func TestStacktraceGoroutine(t *testing.T) {
 		mainCount := 0
 
 		for _, g := range gs {
-			locations, _ := p.GoroutineStacktrace(g, 40)
+			_, locations, _ := p.GoroutineStacktrace(g, 40)
```

Comment:
This seems like an error from a previous commit, but we're asserting no error, but never assigning `err` in the call to `GoroutineStacktrace`.